assets/js/hooks/emoji_picker.js

```diff
@@ -0,0 +1,23 @@
+import { EmojiButton } from "@joeattardi/emoji-button";
```

Comment:
Looking at npm it looks like this package is deprecated and superseded by `picmo`, any reason not to use it?

Reply:
None, i'll bump and see if it still works without any change

Reply:
The API is a bit [different](https://github.com/joeattardi/picmo#basic-usage), but should be equally straightforward!